### PR TITLE
Noob friendly

### PR DESCRIPTION
**Pull request by MrChico**
Contains more documentation and improved error messages to make klab more friendly to people that aren't @mhhf

### CODE REVIEW

**Comment by mhhf** [comment]
I'm @mhhf 
bu

File: lib/srchandler.js [hunk 1]
```diff
@@ -10,26 +16,28 @@ const evm_i = require("./evm.json");
...
-      last,
-      ...list
-    ],
-    [{}]
-  )
-  .reverse()
-  .slice(1)
-
+const genSrcmapArr = srcmap => {
+    if(!srcmap) {revert(`Contract srcmap not found`)}
```
Comment (mhhf) [inline lib/srchandler.js]: Hmm can we push this check out to make this function side-effect free?!

File: libexec/klab-build [hunk 2]
```diff
@@ -39,6 +39,7 @@ const cmd = docopt(usage, {
 });
 
 const warn          = str => console.log(clc.xterm(214)(str))
+if (!testPath("./config.json")) {revert("No config.json file in directory. Review `klab/examples` folder for inspiration")}
```
Comment (mhhf) [inline libexec/klab-build]: 👍

File: lib/build.js [hunk 3]
```diff
@@ -114,9 +114,22 @@ const toK = str => str
   .replace(/uint\(/g, "#unsigned(")
   .replace(/bool\(/g, "bool2Word(")
 
+const longForm = str => {
```
Comment (mhhf) [inline lib/build.js]: like this :+1:

File: examples/token/out/acts/Token_transfer.act [hunk 4]
```diff
@@ -0,0 +1,20 @@
+behaviour transfer of Token
```
Comment (mhhf) [inline examples/token/out/acts/Token_transfer.act]: lets remove the `examples/<name>/out` directory from git - this should be generated by the user

**Comment by MrChico** [comment]
Addressed your comments @mhhf

**Pull request merged**